src/main/java/org/embulk/util/aws/credentials/AwsCredentials.java

```diff
@@ -67,6 +73,9 @@ private static AWSCredentialsProvider getAWSCredentialsProvider(String prefix, A
                 reject(task.getSessionToken(), sessionTokenOption);
                 reject(task.getProfileFile(), profileFileOption);
                 reject(task.getProfileName(), profileNameOption);
+                reject(task.getAccountId(), accountIdOption);
+                reject(task.getRoleName(), roleNameOption);
+                reject(task.getExternalId(), externalIdOption);
```

Comment:
Do we really need to reject these options? I suppose it's not neccessary.
In case you would like to reject these, please reject also `duration`

Reply:
I think we should add it to keep a new code consistent with the current code. Because duration, sessionName and partition has a default value, we can't reject it.